airflow/api_connexion/exceptions.py

```diff
@@ -152,8 +152,8 @@ def __init__(
         )
 
 
-class AlreadyExists(ProblemException):
-    """Raise when the object already exists."""
+class Conflict(ProblemException):
```

Comment:
Why the new exception class?

Reply:
Given that `AlreadyExists` actually already says _Conflict_ in the message, I’d say let’s just rename it (maybe add a line `AlreadyExists = Conflict` just in case, but even that’s not necessary IMO).